src/components/base/Base.js

```diff
@@ -668,9 +668,15 @@ export class BaseComponent {
     let addIcon = this.ce('span', {
       class: 'glyphicon glyphicon-plus'
     });
-    addButton.appendChild(addIcon);
-    addButton.appendChild(this.text(this.component.addAnother || ' Add Another'));
-    return addButton;
+
+    if(justIcon){
+      addButton.appendChild(addIcon);
+      return addButton;
+    }else{
```

Comment:
Please fix spacing in both if statement and else statement. Our standard is as follows.

```
if (justIcon) {

} else {

}
```